presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -663,4 +665,16 @@ public void testFromUtf8()
         assertInvalidFunction("from_utf8(to_utf8('hello'), 'foo')", INVALID_FUNCTION_ARGUMENT);
         assertInvalidFunction("from_utf8(to_utf8('hello'), 1114112)", INVALID_FUNCTION_ARGUMENT);
     }
+
+    @Test
+    public void testCodepoint()
+    {
+        assertFunction("codepoint('x')", INTEGER, (int) 'x');
```

Comment:
Use 0x78 here for the expected value. It's nice to be explicit in tests so you know exactly what is being tested.